libs/designsystem/src/lib/components/form-field/input/input.component.scss

```
@@ -1,7 +1,14 @@
 @use '../form-field-inputs.shared' as shared;
+@use "~@kirbydesign/core/src/scss/interaction-state";
 @use "~@kirbydesign/core/src/scss/utils";
 
 :host {
+  @include interaction-state.transition(background-color);
+  @include interaction-state.apply-hover($loudness: 's') {
```

Unfortunately `$loudness` won't have any effect when we use `apply-hover()` without the state layer solution, so I think this can be reduced to: 

```suggestion
  @include interaction-state.apply-hover {
```